src/libraries/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.cs

```diff
@@ -408,7 +408,7 @@ private bool MatchPattern(ReadOnlySpan<char> relativePath)
         /// </summary>
         private void NotifyInternalBufferOverflowEvent()
         {
-            _onErrorHandler?.Invoke(this, new ErrorEventArgs(
+            OnError(new ErrorEventArgs(
                     new InternalBufferOverflowException(SR.Format(SR.FSW_BufferOverflow, _directory))));
         }
```

Comment:
Do you want a null check of `_onErrorHandler` before calling `OnError()`?

Reply:
Yup, thanks.